addons/cargo/XEH_postInit.sqf

```diff
@@ -78,28 +76,45 @@ GVAR(vehicleAction) = [
     }
 ] call EFUNC(interact_menu,createAction);
 
-GVAR(objectAction) = [
-    QGVAR(load), localize LSTRING(loadObject), "a3\ui_f\data\IGUI\Cfg\Actions\loadVehicle_ca.paa",
-    {
-        params ["_target", "_player"];
-        [_player, _target] call FUNC(startLoadIn);
-    },
-    {
-        //IGNORE_PRIVATE_WARNING ["_target", "_player"];
-        GVAR(enable) &&
-        {(_target getVariable [QGVAR(canLoad), getNumber (configOf _target >> QGVAR(canLoad))]) in [true, 1]} &&
-        {locked _target < 2} &&
-        {alive _target} &&
-        {[_player, _target, ["isNotSwimming"]] call EFUNC(common,canInteractWith)} &&
-        {((nearestObjects [_target, GVAR(cargoHolderTypes), (MAX_LOAD_DISTANCE + 10)]) findIf {
-            private _hasCargoConfig = 1 == getNumber (configOf _x >> QGVAR(hasCargo));
-            private _hasCargoPublic = _x getVariable [QGVAR(hasCargo), false];
-            (_hasCargoConfig || {_hasCargoPublic}) && {_x != _target} && {alive _x} && {locked _x < 2} &&
-            {([_target, _x] call EFUNC(interaction,getInteractionDistance)) < MAX_LOAD_DISTANCE}
-        }) > -1}
-    },
-    LINKFUNC(addCargoVehiclesActions)
-] call EFUNC(interact_menu,createAction);
+GVAR(objectActions)= [
+    [QGVAR(renameObject), localize LSTRING(renameObject), "", //TODO: add icon, maybe a pencil couldn't find it before.
+        {
+            params ["_target"];
+            [_target] spawn FUNC(renameObject);
```

Comment:
Why `spawn`?

Reply:
When using call it will crash the game, not entirely sure why. Spawn seems to not let it crash.

Reply:
Use `CBA_fnc_execNextFrame`. I think it's crashing in unscheduled because interaction menu closes in the same frame as you are trying to create your rename dialog.

Reply:
Yea was my thought as well, will try.